http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -125,16 +126,18 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         kubernetesNamespace
       )
 
-      maybeCromwellDatabaseNames <- maybeCreateCromwellDatabases(app,
-                                                                 params.workspaceId,
-                                                                 params.landingZoneResources,
-                                                                 kubernetesNamespace
+      // If configured for the app type, calls WSM to create a managed identity and postgres databases
+      // for CROMWELL and CROMWELL_RUNNER_APP app types
+      maybeSharedCromwellDatabaseNames <- maybeCreateCromwellDatabases(app,
```

Comment:
I don't think we should call these "shared" since they are per-user, even if they are located in the shared LZ postgres instance

Reply:
Yeah I think `shared` in the name is causing some confusion. What I wanted the variable to represent was that it could have either Cromwell or Cromwell Runner databases in it since for both apps we use `maybeCreateCromwellDatabases` method. I will probably revert the name to `maybeCromwellDatabaseNames` only.